src/containers/LoginPage.tsx

```diff
@@ -28,13 +29,13 @@ const LogoStyled = styled.img`
   margin-bottom: 20px;
 `;
 
-function LoginPage({history}) {
-  const dispatch = useDispatch();
+function LoginPage({history}: any) {
```

Comment:
Try no to use `any` type please. It's ok for now but when you'll gonna fill up the project with code - please make sure you are using not `any` types.